README.md

```diff
@@ -1,117 +1,84 @@
+buckets, and caches using `kubectl`. Deploy it standalone to manage Kubernetes
+clusters, the workloads running on them, and the cloud infrastructure those
+workloads depend upon.
 
-# Overview
+## Architecture
 
-Crossplane is an open source multicloud control plane to manage your
-cloud-native applications and infrastructure across environments, clusters,
-regions and clouds. It enables provisioning and full-lifecycle management
- of applications and managed services from your choice of cloud using `kubectl`.
+![Architecture diagram](docs/media/crossplane-overview.png)
 
-Crossplane can be installed into an existing Kubernetes cluster to add managed
-service provisioning or deployed as a dedicated control plane for multi-cluster
-management and workload scheduling.
+Crossplane builds on the Kubernetes control plane. It is composed of [Stacks] -
+easy to install packages of Kubernetes [custom resources and controllers] that
+extend Crossplane with new functionality. A stack can extend Crossplane with the
+ability to manage the infrastructure of a new cloud provider, or to deploy and
+manage a new cloud-native application. This enables Crossplane users to:
 
-Crossplane enables the community to build and publish Stacks to add more clouds
-and cloud services to Crossplane with support for out-of-tree extensibility and
-independent release schedules. Crossplane includes Stacks for [GCP][stack-gcp], 
-[AWS][stack-aws], and [Azure][stack-azure] today.
+1. Install the **infrastructure stacks** for their desired clouds to enable
+   on-demand, **cloud-portable provisioning of infrastructure** services like
+   databases, caches, and Kubernetes clusters.
+1. Define **portable application workloads**, including their cloud
+   infrastructure dependencies, and schedule them to the most appropriate
+   infrastructure across any cloud.
+1. Package their workloads as an **application stack** that others may easily
+   deploy.
 
-<h4 align="center"><img src="docs/media/crossplane-overview.png" alt="Crossplane"></h4>
+The full vision and architecture of the Crossplane project is described in our
+[architecture document].
 
-Crossplane has four main feature areas that can be used independently:
-1. [Crossplane Services](docs/README.md#crossplane-services) - provision managed services from kubectl.
-1. [Crossplane Stacks](docs/README.md#crossplane-stacks) - extend Crossplane with new functionality.
-1. [Crossplane Workloads](docs/README.md#crossplane-workloads) - define complete applications and schedule across
-   clusters, regions, and clouds.
-1. [Crossplane Clusters](docs/README.md#crossplane-clusters) - manage multiple Kubernetes clusters from a single
-   control plane.
+## Get Started
 
-## Architecture and Vision
+Take a look at the [getting started] guide to learn how to install Crossplane,
+install a stack, and provision cloud infrastructure using `kubectl`. Refer to
+the Crossplane [documentation] for comprehensive guides to using Crossplane,
```

Comment:
```suggestion
the Crossplane [documentation] for comprehensive guides for using Crossplane,
```

Reply:
I believe "guides to using" is more correct in this context.

Reply:
Fair enough! :)